app/src/main/java/org/liberty/android/fantastischmemo/dao/CardDaoImpl.java

```diff
@@ -858,5 +869,69 @@ private void maintainOrdinal() throws SQLException {
             throw new RuntimeException(e);
         }
     }
-}
+
+    public void shuffleCards(){
+        List<Card> cards = queryForAll();
+
+        Card[] cs = cards.toArray(new Card[cards.size()]);
+        Random randomNum = new Random();
+        Card temp;
+        int newNum;
```

Comment:
The AOSP standards say that local variables have to be declared at the point of first use and have an initializer